Formula/libgsm.rb

```diff
@@ -31,8 +31,10 @@ def install
     man1.mkpath
     man3.mkpath
 
+    inreplace "Makefile", "libgsm.1.0.13.dylib", "libgsm.1.0.16.dylib"
```

Comment:
Original Makefile by upstream don't have string of "libgsm.1.0.13.dylib".
It was written by the Patch. And Upstream said that They won't provide _dylib_.